lib/manager/composer/__snapshots__/artifacts.spec.ts.snap

```diff
@@ -121,19 +121,19 @@ Array [
 exports[`.updateArtifacts() supports docker mode 1`] = `
```

Comment:
Would a PR title like `feat(composer): dynamic installation` or `feat(composer): dynamic install` be better.

`instal` is not spelt correctly in any case. 😉